tools/nixpkgs-hammer

```diff
@@ -201,9 +201,14 @@ def render_code_location_ansi(loc: SourceLocation) -> str:
         line_spaces = " " * len(str(loc.line))
         column = loc.column if loc.column is not None else 1
         pointer = " " * (column - 1) + "^"
+        rendered_loc = ":".join(
+            (loc.file, str(loc.line)) + (str(loc.column),)
+            if loc.column is not None
+            else ()
```

Comment:
This reads wrong to me.

And apparently, Python uses this wrong association:

```
>>> 3 + 2 if False else 1
1
```

Reply:
I rewrote it to be procedural.